### ISSUE TITLE
Duplicated parenthesized part

### ISSUE THREAD

**Issue by CTimmerman** [web-scrobbler/web-scrobbler#2753]
**How to reproduce**
1. Go to https://www.youtube.com/watch?v=A6xkXdYOOmI&list=RDCLAK5uy_mk6AmqcHgCRhyJuYsQz5CCVdCF4SRGivs&index=7
4. See "White Room (feat. Jacoby Shaddix) (feat. Jacoby Shaddix)" by "Apocalyptica".

**Expected behavior**
See "White Room (feat. Jacoby Shaddix)" by "Apocalyptica".

<details>
  <summary><b>Debug logs</b> (Click to expand)</summary>

```
util.js:40 Tab 2493: Song set as now playing
util.js:40 Tab 2493: New song detected: {
  "track": "Yorktown",
  "artist": "BUTCHER BABIES",
  "album": null,
  "albumArtist": null,
  "uniqueID": "PkcL2MyUigU",
  "duration": 247,
  "currentTime": 1,
  "isPlaying": true,
  "trackArt": null,
  "isPodcast": false,
  "originUrl": "https://www.youtube.com/watch?v=PkcL2MyUigU&list=RDCLAK5uy_mk6AmqcHgCRhyJuYsQz5CCVdCF4SRGivs&index=6"
}
scrobble-service.js:97 Send "get info" request: 1
util.js:40 Last.fm: track.getinfo response:
{
  "track": {
    "name": "Yorktown",
    "url": "https://www.last.fm/music/Butcher+Babies/_/Yorktown",
    "duration": "0",
    "streamable": {
      "#text": "0",
      "fulltrack": "0"
    },
    "listeners": "2152",
    "playcount": "4401",
    "artist": {
      "name": "Butcher Babies",
      "mbid": "e43b7e0e-0927-4a66-8cd4-8a6bfb162bd8",
      "url": "https://www.last.fm/music/Butcher+Babies"
    },
    "userplaycount": "0",
    "userloved": "0",
    "toptags": {
      "tag": []
    }
  }
}
coverartarchive.org/release/acb9998a-f7de-429a-bfe6-319ae5b0737b/front-500:1 Failed to load resource: the server responded with a status of 404 (NOT FOUND)
coverartarchive.org/release/150aa97d-9c17-4ded-9d5a-403703dbf0f7/front-500:1 Failed to load resource: the server responded with a status of 404 (NOT FOUND)
util.js:40 Tab 2493: Song finished processing: {
  "parsed": {
    "track": "Yorktown",
    "artist": "BUTCHER BABIES",
    "album": null,
    "albumArtist": null,
    "uniqueID": "PkcL2MyUigU",
    "duration": 247,
    "currentTime": 1,
    "isPlaying": true,
    "trackArt": null,
    "isPodcast": false,
    "originUrl": "https://www.youtube.com/watch?v=PkcL2MyUigU&list=RDCLAK5uy_mk6AmqcHgCRhyJuYsQz5CCVdCF4SRGivs&index=6"
  },
  "processed": {
    "track": "Yorktown",
    "artist": "Butcher Babies",
    "albumArtist": null,
    "duration": null
  },
  "flags": {
    "isScrobbled": false,
    "isCorrectedByUser": false,
    "isValid": true,
    "isMarkedAsPlaying": false,
    "isSkipped": false,
    "isReplaying": false
  },
  "metadata": {
    "userloved": false,
    "startTimestamp": 1615025099,
    "label": "YouTube",
    "artistUrl": "https://www.last.fm/music/Butcher+Babies",
    "trackUrl": "https://www.last.fm/music/Butcher+Babies/_/Yorktown",
    "userPlayCount": 0
  },
  "connectorLabel": "YouTube"
}
util.js:40 Tab 2493: The song will be scrobbled in 124 seconds
util.js:40 Tab 2493: The song will be repeated in 247 seconds
scrobble-service.js:113 Send "now playing" request: 1
util.js:40 Last.fm: track.updatenowplaying response:
{
  "nowplaying": {
    "artist": {
      "corrected": "0",
      "#text": "Butcher Babies"
    },
    "ignoredMessage": {
      "code": "0",
      "#text": ""
    },
    "album": {
      "corrected": "0",
      "#text": ""
    },
    "albumArtist": {
      "corrected": "0",
      "#text": ""
    },
    "track": {
      "corrected": "0",
      "#text": "Yorktown"
    }
  }
}
util.js:40 Tab 2493: Song set as now playing
inject.js:52 Injecting vendor/filter.js
inject.js:52 Injecting vendor/jquery.min.js
inject.js:52 Injecting vendor/browser-polyfill.min.js
inject.js:52 Injecting core/content/util.js
inject.js:52 Injecting core/content/reactor.js
inject.js:52 Injecting core/content/connector.js
inject.js:52 Injecting connectors/youtube.js
inject.js:52 Injecting core/content/starter.js
util.js:40 Tab 2573: Created controller for YouTube connector
inject.js:52 Injecting vendor/filter.js
inject.js:52 Injecting vendor/jquery.min.js
inject.js:52 Injecting vendor/browser-polyfill.min.js
inject.js:52 Injecting core/content/util.js
inject.js:52 Injecting core/content/reactor.js
inject.js:52 Injecting core/content/connector.js
inject.js:52 Injecting connectors/youtube.js
inject.js:52 Injecting core/content/starter.js
util.js:40 Tab 2574: Created controller for YouTube connector
inject.js:52 Injecting vendor/filter.js
inject.js:52 Injecting vendor/jquery.min.js
inject.js:52 Injecting vendor/browser-polyfill.min.js
inject.js:52 Injecting core/content/util.js
inject.js:52 Injecting core/content/reactor.js
inject.js:52 Injecting core/content/connector.js
inject.js:52 Injecting connectors/youtube.js
inject.js:52 Injecting core/content/starter.js
util.js:40 Tab 2575: Created controller for YouTube connector
inject.js:52 Injecting vendor/filter.js
inject.js:52 Injecting vendor/jquery.min.js
inject.js:52 Injecting vendor/browser-polyfill.min.js
inject.js:52 Injecting core/content/util.js
inject.js:52 Injecting core/content/reactor.js
inject.js:52 Injecting core/content/connector.js
inject.js:52 Injecting connectors/youtube.js
inject.js:52 Injecting core/content/starter.js
util.js:40 Tab 2576: Created controller for YouTube connector
inject.js:52 Injecting vendor/filter.js
inject.js:52 Injecting vendor/jquery.min.js
inject.js:52 Injecting vendor/browser-polyfill.min.js
inject.js:52 Injecting core/content/util.js
inject.js:52 Injecting core/content/reactor.js
inject.js:52 Injecting core/content/connector.js
inject.js:52 Injecting connectors/youtube.js
inject.js:52 Injecting core/content/starter.js
util.js:40 Tab 2577: Created controller for YouTube connector
scrobble-service.js:129 Send "scrobble" request: 1
util.js:40 Last.fm: track.scrobble response:
{
  "scrobbles": {
    "@attr": {
      "accepted": 1,
      "ignored": 0
    },
    "scrobble": {
      "artist": {
        "corrected": "0",
        "#text": "Butcher Babies"
      },
      "ignoredMessage": {
        "code": "0",
        "#text": ""
      },
      "albumArtist": {
        "corrected": "0",
        "#text": ""
      },
      "timestamp": "1615025099",
      "album": {
        "corrected": "0"
      },
      "track": {
        "corrected": "0",
        "#text": "Yorktown"
      }
    }
  }
}
util.js:40 Tab 2493: Scrobbled successfully
util.js:40 Tab 2493: isPlaying state changed to false
util.js:40 Tab 2493: Received empty state - resetting
util.js:40 Tab 2493: New song detected: {
  "track": "White Room",
  "artist": "Shaddix",
  "album": null,
  "albumArtist": null,
  "uniqueID": "PkcL2MyUigU",
  "duration": 246,
  "currentTime": null,
  "isPlaying": true,
  "trackArt": null,
  "isPodcast": false,
  "originUrl": "https://www.youtube.com/watch?v=A6xkXdYOOmI&list=RDCLAK5uy_mk6AmqcHgCRhyJuYsQz5CCVdCF4SRGivs&index=7"
}
scrobble-service.js:97 Send "get info" request: 1
util.js:40 Tab 2493: New song detected: {
  "track": "White Room (feat. Jacoby Shaddix) (feat. Jacoby Shaddix)",
  "artist": "Apocalyptica",
  "album": null,
  "albumArtist": null,
  "uniqueID": "A6xkXdYOOmI",
  "duration": 246,
  "currentTime": null,
  "isPlaying": true,
  "trackArt": null,
  "isPodcast": false,
  "originUrl": "https://www.youtube.com/watch?v=A6xkXdYOOmI&list=RDCLAK5uy_mk6AmqcHgCRhyJuYsQz5CCVdCF4SRGivs&index=7"
}
scrobble-service.js:97 Send "get info" request: 1
util.js:40 Last.fm: track.getinfo response:
{
  "error": 6,
  "message": "Track not found",
  "links": []
}
scrobble-service.js:101 Unable to get song info from Last.fm
(anonymous) @ scrobble-service.js:101
util.js:40 Last.fm: track.getinfo response:
{
  "error": 6,
  "message": "Track not found",
  "links": []
}
scrobble-service.js:101 Unable to get song info from Last.fm
(anonymous) @ scrobble-service.js:101
coverartarchive.org/release/34d831b9-e125-4a86-9f49-ea0dc068f630/front-500:1 Failed to load resource: the server responded with a status of 404 (NOT FOUND)
coverartarchive.org/release/f7ce2cac-cc49-4fe0-92cc-4450cccc2698/front-500:1 Failed to load resource: the server responded with a status of 404 (NOT FOUND)
util.js:40 Tab 2493: Song finished processing: {
  "parsed": {
    "track": "White Room (feat. Jacoby Shaddix) (feat. Jacoby Shaddix)",
    "artist": "Apocalyptica",
    "album": null,
    "albumArtist": null,
    "uniqueID": "A6xkXdYOOmI",
    "duration": 246,
    "currentTime": 1,
    "isPlaying": true,
    "trackArt": null,
    "isPodcast": false,
    "originUrl": "https://www.youtube.com/watch?v=A6xkXdYOOmI&list=RDCLAK5uy_mk6AmqcHgCRhyJuYsQz5CCVdCF4SRGivs&index=7"
  },
  "processed": {
    "track": null,
    "album": null,
    "artist": null,
    "albumArtist": null,
    "duration": null
  },
  "flags": {
    "isScrobbled": false,
    "isCorrectedByUser": false,
    "isValid": false,
    "isMarkedAsPlaying": false,
    "isSkipped": false,
    "isReplaying": false
  },
  "metadata": {
    "startTimestamp": 1615025346,
    "label": "YouTube"
  },
  "connectorLabel": "YouTube"
}
coverartarchive.js:35 Found album artwork via MusicBrainz
util.js:40 Tab 2493: isPlaying state changed to false
util.js:40 Tab 2494: Remove controller for YouTube connector
util.js:40 Tab 2495: Remove controller for YouTube connector
util.js:40 Tab 2496: Remove controller for YouTube connector
util.js:40 Tab 2512: Remove controller for YouTube connector
util.js:40 Tab 2513: Remove controller for YouTube connector
util.js:40 Tab 2523: Remove controller for YouTube connector
util.js:40 Tab 2524: Remove controller for YouTube connector
util.js:40 Tab 2528: Remove controller for YouTube connector
util.js:40 Tab 2565: Remove controller for YouTube connector
util.js:40 Tab 2567: Remove controller for YouTube connector
util.js:40 Tab 2568: Remove controller for YouTube connector
util.js:40 Tab 2569: Remove controller for YouTube connector
util.js:40 Tab 2570: Remove controller for YouTube connector
util.js:40 Tab 2573: Remove controller for YouTube connector
util.js:40 Tab 2574: Remove controller for YouTube connector
util.js:40 Tab 2575: Remove controller for YouTube connector
util.js:40 Tab 2576: Remove controller for YouTube connector
util.js:40 Tab 2577: Remove controller for YouTube connector
util.js:40 Tab 2493: isPlaying state changed to true
util.js:40 Tab 2493: Song unprocessed: {
  "parsed": {
    "track": "White Room (feat. Jacoby Shaddix) (feat. Jacoby Shaddix)",
    "artist": "Apocalyptica",
    "album": null,
    "albumArtist": null,
    "uniqueID": "A6xkXdYOOmI",
    "duration": 246,
    "currentTime": 48,
    "isPlaying": true,
    "trackArt": null,
    "isPodcast": false,
    "originUrl": "https://www.youtube.com/watch?v=A6xkXdYOOmI&list=RDCLAK5uy_mk6AmqcHgCRhyJuYsQz5CCVdCF4SRGivs&index=7"
  },
  "processed": {
    "track": null,
    "album": null,
    "artist": null,
    "albumArtist": null,
    "duration": null
  },
  "flags": {
    "isScrobbled": false,
    "isCorrectedByUser": false,
    "isValid": false,
    "isMarkedAsPlaying": false,
    "isSkipped": false,
    "isReplaying": false
  },
  "metadata": {
    "startTimestamp": 1615025346,
    "label": "YouTube"
  },
  "connectorLabel": "YouTube"
}
util.js:40 Tab 2493: Clearing playback timer destination time
scrobble-service.js:97 Send "get info" request: 1
util.js:40 Last.fm: track.getinfo response:
{
  "track": {
    "name": "White Room (feat. Jacoby Shaddix)",
    "url": "https://www.last.fm/music/Apocalyptica/_/White+Room+(feat.+Jacoby+Shaddix)",
    "duration": "0",
    "streamable": {
      "#text": "0",
      "fulltrack": "0"
    },
    "listeners": "1578",
    "playcount": "2114",
    "artist": {
      "name": "Apocalyptica",
      "mbid": "788ad31c-bf0c-4a31-83f8-b8b130d79c76",
      "url": "https://www.last.fm/music/Apocalyptica"
    },
    "userplaycount": "0",
    "userloved": "0",
    "toptags": {
      "tag": []
    }
  }
}
coverartarchive.org/release/37c8ae0b-934f-4c28-a107-be4a969b894c/front-500:1 Failed to load resource: the server responded with a status of 404 (NOT FOUND)
coverartarchive.org/release/84cd6b68-f941-4fb4-add9-849b54a7196e/front-500:1 Failed to load resource: the server responded with a status of 404 (NOT FOUND)
util.js:40 Tab 2493: Song finished processing: {
  "parsed": {
    "track": "White Room (feat. Jacoby Shaddix) (feat. Jacoby Shaddix)",
    "artist": "Apocalyptica",
    "album": null,
    "albumArtist": null,
    "uniqueID": "A6xkXdYOOmI",
    "duration": 246,
    "currentTime": 49,
    "isPlaying": true,
    "trackArt": null,
    "isPodcast": false,
    "originUrl": "https://www.youtube.com/watch?v=A6xkXdYOOmI&list=RDCLAK5uy_mk6AmqcHgCRhyJuYsQz5CCVdCF4SRGivs&index=7"
  },
  "processed": {
    "track": "White Room (feat. Jacoby Shaddix)",
    "album": null,
    "artist": "Apocalyptica",
    "albumArtist": null,
    "duration": null
  },
  "flags": {
    "isScrobbled": false,
    "isCorrectedByUser": true,
    "isValid": true,
    "isMarkedAsPlaying": false,
    "isSkipped": false,
    "isReplaying": false
  },
  "metadata": {
    "userloved": false,
    "startTimestamp": 1615031304,
    "label": "YouTube",
    "artistUrl": "https://www.last.fm/music/Apocalyptica",
    "trackUrl": "https://www.last.fm/music/Apocalyptica/_/White+Room+(feat.+Jacoby+Shaddix)",
    "userPlayCount": 0
  },
  "connectorLabel": "YouTube"
}
util.js:40 Tab 2493: The song will be scrobbled in 73 seconds
util.js:40 Tab 2493: The song will be repeated in 246 seconds
scrobble-service.js:113 Send "now playing" request: 1
util.js:40 Last.fm: track.updatenowplaying response:
{
  "nowplaying": {
    "artist": {
      "corrected": "0",
      "#text": "Apocalyptica"
    },
    "ignoredMessage": {
      "code": "0",
      "#text": ""
    },
    "album": {
      "corrected": "0",
      "#text": ""
    },
    "albumArtist": {
      "corrected": "0",
      "#text": ""
    },
    "track": {
      "corrected": "0",
      "#text": "White Room (feat. Jacoby Shaddix)"
    }
  }
}
util.js:40 Tab 2493: Song set as now playing
scrobble-service.js:150 Send "love" request: 2
scrobble-service.js:84 Libre.fm is not bound
unbindScrobbler @ scrobble-service.js:84
util.js:40 Last.fm: track.love response:
{}
scrobble-service.js:129 Send "scrobble" request: 1
util.js:40 Last.fm: track.scrobble response:
{
  "scrobbles": {
    "@attr": {
      "accepted": 1,
      "ignored": 0
    },
    "scrobble": {
      "artist": {
        "corrected": "0",
        "#text": "Apocalyptica"
      },
      "ignoredMessage": {
        "code": "0",
        "#text": ""
      },
      "albumArtist": {
        "corrected": "0",
        "#text": ""
      },
      "timestamp": "1615031304",
      "album": {
        "corrected": "0"
      },
      "track": {
        "corrected": "0",
        "#text": "White Room (feat. Jacoby Shaddix)"
      }
    }
  }
}
util.js:40 Tab 2493: Scrobbled successfully
DevTools failed to load SourceMap: Could not load content for chrome-extension://hhinaapppaileiechjoiifaancjggfjm/vendor/md5.min.js.map: HTTP error: status code 404, net::ERR_UNKNOWN_URL_SCHEME
```
</details>

**Environment (please complete the following information)**
-   OS: Windows 10
-   Browser: Chrome
-   Extension version: 2.41.0

**Comment by CTimmerman** [comment]
The second one appears to be generated from the description text: https://www.youtube.com/watch?v=qFiGYtX4ey0 adds the guitarists as featured for some reason.

**Comment by alexesprit** [comment]
Aha, the extension adds featuring artists even if they already listed in the song title.

**Comment by alexesprit** [comment]
Closed via 019478016bc75560fad7937a46748b04ea47df4e.
